android/capacitor/src/main/java/com/getcapacitor/cordova/MockCordovaInterfaceImpl.java

```diff
@@ -14,4 +16,23 @@ public MockCordovaInterfaceImpl(Activity activity) {
     public CordovaPlugin getActivityResultCallback() {
         return this.activityResultCallback;
     }
+
+    /**
+     * Use in lieu of {@link CordovaInterfaceImpl#onRequestPermissionResult(int, String[], int[])} to return
```

Comment:
Is `CordovaInterfaceImpl#onRequestPermissionResult()` still used? Pretty sure we can remove it.

Reply:
That's defined on the parent class in Apache's library, and I don't see it referenced anywhere else now so I can just clean up this comment to avoid confusion.